eip-1175.md

```diff
@@ -0,0 +1,529 @@
+---
+eip: 1175
+title: Wallet & shop standard for all tokens (erc20)
```

Comment:
Please add a discussions-to URL with a public, permanent URL where people can comment on this EIP.